traceui/grafana.ini

```diff
@@ -0,0 +1,3 @@
+[users]
```

Comment:
This file should probably enable the feature toggle.

Reply:
Was actually planning to remove the directory or put it into a separate PR.

Reply:
https://github.com/grafana/grafana/pull/21029